pkg/cmd/resource/events.go

```diff
@@ -0,0 +1,31 @@
+package resource
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/stripe/stripe-cli/pkg/config"
+)
+
+// AddEventsSubCmds adds custom subcommands to the `events` command created
+// automatically as a resource command.
+func AddEventsSubCmds(rootCmd *cobra.Command, cfg *config.Config) error {
+	found := false
+
+	for _, cmd := range rootCmd.Commands() {
+		if cmd.Use == "events" {
+			found = true
+
+			NewEventsResendCmd(cmd, cfg)
+
+			break
+		}
+	}
+
+	if !found {
```

Comment:
Is this mostly for people who develop on the cli? Seems like it won't happen during runtime.

Reply:
Yep, exactly.